simple_ai_agents/newsletter_agent/.env.example

```
@@ -1,2 +1,43 @@
+# Copy this file to .env and fill in your actual values
+# IMPORTANT: Never commit .env files to version control
+
+# =============================================================================
+# Required Configuration
+# =============================================================================
+
+# Nebius AI API Key (Required)
+# Description: Primary LLM provider for the application
+# Get your key: https://studio.nebius.ai/api-keys
+# Free tier: 100 requests/minute
+NEBIUS_API_KEY="your_nebius_api_key_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)